setup/src/Magento/Setup/Console/Command/AdminUserCreateCommand.php

```diff
@@ -96,20 +97,20 @@ public function getOptionsList()
     /**
      * Check if all admin options are provided
      *
-     * @param InputInterface $input
+     * @param string[] $input
      * @return string[]
      */
-    public function validate(InputInterface $input)
+    public function validate(array $data)
```

Comment:
We can do anything if we set our minds to it :stuck_out_tongue: 

Yes, I'm having a rethink on my implementation that will also address @buskamuza's array access comment.